API/README.md

```diff
@@ -0,0 +1,2 @@
+# API Test - Postman
```

Comment:
If you don't have instructions here, you can remove from this PR and put instructions in a separate PR later.